cmd/ci-operator-prowgen/main.go

```diff
@@ -57,9 +57,8 @@ exec ci-operator \
     --target=%[1]s \
     --template=/tmp/%[1]s
 `
-
-	sshKeyPath    = "/usr/local/github-ssh-credentials-openshift-bot"
-	sshSecretName = "github-ssh-credentials-openshift-bot"
+	oauthSecretName = "oauth-token"
+	oauthKey        = "oauth"
```

Comment:
Let's use a better-named, appropriately scoped token for this purpose. I'll prep it tomorrow together with a `populate...` script change.

Reply:
I initially opened this PR to add the OAuth authentication but then I realize that I need make `ci-operator` first to support this. 

Those names are supposed to point to `oauth-token` secret in `ci` namespace and the `oauth` is the key. 
However the approach now is changed, since we only need to add those under `volumes/volumeMounts` and not in `DecorationConfig`

Reply:
> When you update the secret, take care also the prow components that are using that.

That's the point, I do not want to change prow components; let them use whatever secrets they are using: they need all kinds of permissions. But here I want to use a properly named secret containing a token with properly scoped permissions. The jobs only need to clone code, so the token only needs read rights on repos.

Reply:
Yea, that makes sense. We need a new token for cloning as you said. Thanks for pointing this out.

Reply:
Once https://github.com/openshift/release/pull/6339 is merged we can use the following in Prowgen

Secret name: `github-credentials-openshift-ci-robot-private-git-cloner`, item `oauth`

Reply:
We should not be using the `oauth-token` secret

```
$ grep 'oauth-token' ci-operator/populate-secrets-from-bitwarden.sh
$
```

All places you see it used need to change to use the appropriate `github-credentials-` secret, please.